packages/compartment-mapper/src/policy.js

```diff
@@ -154,20 +131,14 @@ export const getPolicyForPackage = (namingKit, policy) => {
   if (canonicalName === ATTENUATORS_COMPARTMENT) {
     return {
       defaultAttenuator: policy.defaultAttenuator,
-      packages: detectAttenuators(policy).reduce((packages, specifier) => {
-        packages[specifier] = true;
-        return packages;
-      }, {}),
+      packages: 'any',
```

Comment:
this removed section had the bug where we mistook specifier for a resource id. and it was all unnecessary.

Reply:
packages is normally a key/value map. there's only one string value allowed and that's the "any". There's no room for a mistake here